website/docs/docs/collaborate/model-query-history.md

```diff
@@ -46,6 +44,7 @@ To enable model query history in dbt Cloud, follow these steps:
 2. Select the environment marked **PROD** and click **Settings**.
 3. Click **Edit** and scroll to the **Query History** section to enable the query history toggle. When it’s green and to the right, it's enabled.
 4. Click the **Test Permissions** button to validate the deployment credentials permissions are sufficient to support query history.
+5. dbt Cloud automatically enables query history for brand new environments. If query history fails to retrieve data, dbt Cloud disables it to prevent unintended warehouse costs. You’ll need to manually re-enable it once you resolve the issue.
```

Comment:
```suggestion
5. dbt Cloud automatically enables query history for brand new environments. If query history fails to retrieve data, dbt Cloud automatically disables it to prevent unintended warehouse costs. To re-enable it, please reach out to [dbt Support](mailto:support@getdbt.com).
```